pubspec.yaml

```diff
@@ -1,8 +1,8 @@
 name: very_good_infinite_list
 description: >-
-  A Very Good Infinite List Widget created by Very Good Ventures.
+  A library for easily displaying paginated data, created by Very Good Ventures.
   Comes in handy when making activity feeds, news feeds, etc.
-version: 0.3.0
+version: 1.0.0
```

Comment:
thoughts on 0.4.0 haha 😅 ? I want to give some more time to let the package soak before calling it stable.

Reply:
Fair enough 😄